src/NzbDrone.Core/Datastore/Migration/033_download_propers_config.cs

```diff
@@ -0,0 +1,43 @@
+using System.Data;
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(033)]
+    public class download_propers_config : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.WithConnection(SetMetadataFileExtension);
+            Execute.Sql("DELETE FROM Config WHERE Key = 'autodownloadpropers'");
+        }
+
+        private void SetMetadataFileExtension(IDbConnection conn, IDbTransaction tran)
```

Comment:
Is this function name just a copy-paste from elsewhere?  Not sure it's doing anything with metadata extensions?

Reply:
Yep,, my bad